Gemfile

```diff
@@ -13,4 +14,4 @@ gem 'rspec'
 gem 'rubocop', '~> 0.73.0'
 gem 'rubocop-rspec'
 gem 'selenium-webdriver'
-gem 'sdr-client', '~> 0.16.0'
+gem 'sdr-client', '~> 0.16'
```

Comment:
@jcoyne OK to relax this?

Reply:
erg - isn't this how we end up breaking a ton of apps at once if a pre 1.0 release has backward breaking changes?

Reply:
that's when we pin DSC to the minor of cocina-models, I think. I wasn't too worried about our integration tests using the latest and greatest.

Reply:
oh duh.  reviewing too many PRs at once.